README.md

```diff
@@ -16,24 +16,37 @@ There three categories of metrics that Moneo monitors:
     - Power
     - ECC Counts
     - GPU Throttling
-2.	GPU Profiling Counters
+1.	GPU Profiling Counters
     - SM Activity
     - Memory Dram Activity
     - NVLink Activity
     - PCIE Rate
-3.	InfiniBand Network Counters
+1.	InfiniBand Network Counters
     - IB TX/RX rate
     - IB Port errors
+</details>
+
+<details>
+<summary>Grafana Dashboards</summary>
+
+1. Menu: List of available dashboards.
+![image](https://user-images.githubusercontent.com/70273488/186491552-34d545de-b60d-4cdd-aed4-702e2f34d974.png)
+
+   Note: When viewing GPU dashboards make sure to note whether you are using Nvidia or AMD GPU nodes and select the proper dashboard.
 
-### _Views_
-1. Device Counters
+2. Cluster View: contains  min, max, average across devices for GPU/IB metrics per VM.
+![image](https://user-images.githubusercontent.com/70273488/186491238-ce032f87-cab1-41c4-b32e-9826b8c47b72.png)
```

Comment:
Why does the first image have values for "Max" and "Last" ? Should it be "Max" and "Min"?

Reply:
Are you referencing the specific Graphs on the Cluster wide view?

Reply:
Yeah, this image. Maybe add a description of this image to the text? 

https://user-images.githubusercontent.com/70273488/186491238-ce032f87-cab1-41c4-b32e-9826b8c47b72.png

Reply:
Yea those were chosen because they show max and current(last) values. This can give the user some insight in to current perf and how it relates to peak/max perf.

Min would pretty much always show 0 (unless homed in on a specific time interval). At least this is my reasoning